ibis/expr/operations.py

```diff
@@ -516,6 +521,53 @@ class Log10(Logarithm):
     """Logarithm base 10"""
 
 
+# TRIGONOMETRIC OPERATIONS
+
+class TrigonometryUnary(UnaryOp):
```

Comment:
Can you change the name from `TrigonometryUnary` to `TrigonometricUnary`, and do the same for `TrigonometryBinary`.